collectd-jenkins/README.md

```diff
@@ -0,0 +1,229 @@
+
+    [<img src='./img/chart-jenkins-vm-memory-utilization.png' width=200px>](./img/chart-jenkins-vm-memory-utilization.png)
+
+  - **Heap Usage**: Line graph of the utilization percentage of Heap memory by each Jenkins instance.
+
+    [<img src='./img/chart-jenkins-cluster-append-recv.png' width=200px>](./img/chart-jenkins-cluster-append-recv.png)
+
+  - **Non-Heap Used**: Line graph of the non-heap memory used by each Jenkins instance.
+
+    [<img src='./img/chart-jenkins-non-heap-used.png' width=200px>](./img/chart-jenkins-non-heap-used.png)
+
+- **Jenkins Master**:
+
+  - **Top 5 Failed Jobs**: Shows the top 5 failed jobs over the past day based on the total failure count in an instance(s).
+    
+    [<img src='./img/chart-jenkins-master-top-5-failed-jobs.png' width=200px>](./img/chart-jenkins-master-top-5-failed-jobs.png)
+
+  - **Health Checks**: The status of each health check as reported by DropWizard Metrics. This gives a quick overview of what's wrong with the instance.
+
+    [<img src='./img/chart-jenkins-master-health-checks.png' width=200px>](./img/chart-jenkins-master-health-checks.png)
+
+  - **Slave Status**: Shows the number of slave agents of the instance(s) that are alive.
+  
+      [<img src='./img/chart-jenkins-master-slave-status.png' width=200px>](./img/chart-jenkins-master-slave-status.png)
+
+  - **Busy Executors vs Pending Jobs**: A line chart showing comparison between in-use executors and pending jobs in queue in an instance(s). On comparing this chart with two above, reason for job failures can be narrowed down further quickly.
+  
+      [<img src='./img/chart-jenkins-master-busy-executors-vs-pending-jobs.png' width=200px>](./img/chart-jenkins-busy-executors-vs-pending-jobs.png)
+
+  - **System CPU Utilization** A line graph showing system CPU utilization by a particular host.
+
+    [<img src='./img/chart-jenkins-master-system-cpu-utilization.png' width=200px>](./img/chart-jenkins-master-system-cpu-utilization.png)
+  
+  - **VM Memory Utilization**: Area chart of the memory used by the Jenkins JVM instance(s) on a host.
+  
+      [<img src='./img/chart-jenkins-master-vm-memory-utilization.png' width=200px>](./img/chart-jenkins-vm-memory-utilization.png)
+
+All DropWizard metrics reported by the jenkins collectd plugin will not contain any dimensions by default. Whereas, the job metrics sent will contain the following dimensions by default:
+
+* `Job`, name of the job
+* `Result`, the status of the job
+
+A few other details:
+
+* `plugin` is always set to `jenkins`
+* `plugin_instance` will contain the IP address and the port of the member given in the configuration
+* To add metrics from the ```/metrics/<MetricsKey>/metrics``` endpoint, use the configuration options mentioned in [configuration](#configuration). If metrics are being included individually, make sure to give names that are valid. For example, ```vm.daemon.count``` or ```vm.terminated.count```
```

Comment:
placing "/metrics/<MetricsKey>/metrics", "vm.daemon.count", "vm.terminated.count" inside single "`" would suffice because it's not a code block